lifecycle/src/components/TodoList.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import ToDoItem from "./TodoItem";
+
+class ToDoList extends React.Component {
```

Comment:
since this component doesn't need to have state, it would be better if it just was a functional component